api-report/azure-client.api.md

```diff
@@ -50,12 +50,14 @@ export interface AzureClientProps {
 
 // @public
 export interface AzureConnectionConfig {
-    orderer: string;
-    storage: string;
-    tenantId: "local" | string;
+    endpoint: string;
     tokenProvider: ITokenProvider;
+    type: AzureConnectionConfigType;
 }
 
+// @public (undocumented)
```

Comment:
Nit: documentation please.

Reply:
I see there's a docstring, so maybe you just need to do a local build.

Reply:
done 👍